conftest.py

```diff
@@ -607,7 +607,7 @@ def _upload_cluster_dump(client):
     yield _upload_cluster_dump
 
 
-P2P_AVAILABLE = parse_version(dask.__version__) >= parse_version("2022.11.0")
+P2P_AVAILABLE = Version(dask.__version__) >= Version("2022.11.0")
```

Comment:
Just a heads up, I'm working on a PR ([here](https://github.com/coiled/coiled-runtime/pull/595)) to only test `runtime >=0.2.1` which will cover this case as the dask version is 2022.11.1.